exonum-java-binding/core/src/main/java/com/exonum/binding/core/runtime/DispatcherSchema.java

```diff
@@ -36,8 +36,8 @@ public DispatcherSchema(View view) {
   /**
    * Returns a map of service instance specifications of started services indexed by their names.
    */
-  public MapIndexProxy<String, InstanceSpec> serviceInstances() {
-    return MapIndexProxy.newInstance("core.dispatcher.service_instances", view,
-        string(), protobuf(InstanceSpec.class));
+  public ProofMapIndexProxy<String, InstanceState> serviceInstances() {
+    return ProofMapIndexProxy.newInstance("dispatcher_instances", view,
```

Comment:
No "core" prefix? Just in global namespace?

Reply:
According to https://github.com/exonum/exonum/blob/master/exonum/src/runtime/dispatcher/schema.rs#L31